python/setup.py

```diff
@@ -12,24 +12,25 @@
     readme_description = ""
 
 runtime_dependencies = [
```

Comment:
Future request but maybe we can do this kind of change in its own PR to limit testing dependency issues versus PR issues.